lnrpc/routerrpc/router.proto

```diff
@@ -336,26 +340,31 @@ message NodeHistory {
     /// Time stamp of last failure. Set to zero if no failure happened yet.
     int64 last_fail_time = 2 [json_name = "last_fail_time"];
 
-    /// Estimation of success probability for channels not in the channel array.
-    float other_chan_success_prob = 3 [json_name = "other_chan_success_prob"];
+    /**
+    Estimation of success probability for node pairs that are not in the pairs
+    list.
```

Comment:
should maybe specify that these are pairs where this node is one of them, and the direction of the pair?

Reply:
yes, it is indeed vaguely formulated. changed.